radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/RadixHash.java

```diff
@@ -73,4 +74,6 @@ public int hashCode() {
 	public String toString() {
 		return Base64.toBase64String(hash);
 	}
+
+	public String toHexString() { return ByteString.of(hash).hex(); }
```

Comment:
Would prefer to see this following the coding conventions:
```
public String toHexString() {
    return Bytes.toHexString(hash);
}
````

Reply:
Absolutely, I got confused by Intellij which collapsed e.g. `hashCode` into one line. Got it!